admin-cli/executor/disk_info.go

```diff
@@ -97,6 +99,31 @@ func QueryAllNodesDiskInfo(client *Client, tableName string) (map[string]*radmin
 	return respMap, nil
 }
 
+func QueryAliveNodesDiskInfo(client *Client, tableName string) (map[string]*radmin.QueryDiskInfoResponse, error) {
```

Comment:
Is it related to the subject of this patch?

Reply:
The original QueryNodesDiskInfo was not user-friendly and couldn't handle certain situations, such as missing table partition on nodes or some nodes being unalive.

Reply:
Yes, submit this in another PR would be better. Keep each commit independent.

Reply:
Thinks, I have removed the function.